source/concepts/applications.rst

```diff
@@ -0,0 +1,746 @@
+not applied.
+
+.. _velocity-reset:
+
+Velocity reset
+~~~~~~~~~~~~~~
+
+Some server implementation can kill the momentum from ended user interactions
+by setting the velocity of the affected particles to 0. This is called velocity
+reset and can be requested by the user as part of the interaction description.
+
+Servers that have the ability to do velocity reset should advertise the feature
+by setting the ``imd.velocity_reset_available`` key to true in the :ref:`shared
+state <state-service>`.
+
+Sending user interactions
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Users send, on the :ref:`shared state <state-service>`, the description of the
+interactions they want to apply. There is no limit to the number of interaction
+a user can send. Each interaction is described under the key
+``interaction.<INTERACTION_ID>`` where ``<INTERACTION_ID>`` is an arbitrary
+string, unique to the interaction, used to identify it. It is commonly a UUID4.
+Under that key, the value is a Struct with the following keys:
+
+* ``positions``: the coordinates of the interaction's origin in simulation
+  space. This is typically a position attached to the controller of the user in
+  VR, but it does not have to be. By default, this is `[0, 0, 0]`.
+* ``particles``: the indices of the affected particles in the array of
+  particles used by the :ref:`trajectory application <trajectory-application>`.
+  If the order in this array does not match the order used by the simulation
+  engine, it is the server's responsibility to map them. The default value is
+  an empty list.
+* ``type``: the kind of interaction to apply, this is what defines what
+  :ref:`force equation <force-equations>` will be used. It should be set to
+  `gaussian` for the Gaussian force, `spring` for the harmonic force, and
+  `constant` for the constant force. Interactions with an type unknown to the
+  server will be ignored silently. By default, the Gaussian force is assumed.
+* ``scale``: the scaling factor :math:`s` to apply to the force. The default
+  scale is 1.
+* ``mass_weighted``: a boolean, true if the interaction is mass weighted, false
+  otherwise. The default is true.
+* ``max_force``: the maximum force magnitude that can be applied to a particle
+  by this interaction. The default is 20,000
+  :math:`kJ\cdot\text{mol}^{-1}\cdot\text{nm}^{-1}`.
+* ``reset_velocities``: a boolean, true if :ref:`velocity reset
+  <velocity-reset>` should be applied, false otherwise. This is false by
+  default and will be ignored silently is the server does not have the feature.
+
+If the iMD application is used in conjunction with the :ref:`multiplayer
+application <multiplayer-application>`, then the interaction can also use the
+following fields:
+
+* ``owner.id``: if the interaction originates from a client that defines an
+  avatar, it can set this field to the player id attached to its avatar. This
+  allows to match interactions with avatars when analysing session recordings.
```

Comment:
In this scenario, is it the server or the client that is matching the interactions with the avatars? It may be worth specifying, e.g. 
```suggestion
  allows the server to match interactions with avatars when analysing session recordings.
```
in the former scenario.

Reply:
Perhaps `allows one to...`, unless this sounds too pretentious?